services/order.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+from django.db import transaction
+from db.models import Order, Ticket
+from django.core.exceptions import ValidationError
+from django.contrib.auth import get_user_model
+from datetime import datetime
+
+User = get_user_model()
+
+
+def create_order(tickets: List[dict],
+                 username: str,
+                 date: [str] = None) -> None:
+
+    try:
+        user = User.objects.get(username=username)
```

Comment:
The `create_order` function uses `User.objects.get(username=username)`, which will raise a `User.DoesNotExist` exception if the user is not found. This is correctly handled by raising a `ValidationError`, which is good practice.